.readthedocs.yml

```diff
@@ -20,6 +17,8 @@ formats: all
 
 python:
   install:
-  - method: setuptools
-    path: .
-  - requirements: docs/requirements.in
+    - method: pip
+      path: .
+    - method: setuptools
+      path: .
```

Comment:
This shouldn't be necessary. `pip` will build and install it in the previous step. And at that point, all of the version metadata must already be present in the Git checkout.

Reply:
Ah, wrong diff.